packages/pg/lib/connection-parameters.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 var dns = require('dns')
+const _ = require('lodash')
```

Comment:
Undeclared dependency, and too heavy just to check if an object is empty (should pull in just https://www.npmjs.com/package/lodash.isempty if really necessary).